include/lsst/afw/math/warpExposure.h

```diff
@@ -461,7 +462,7 @@ int warpImage(DestImageT &destImage,                 ///< remapped %image
 template <typename DestImageT, typename SrcImageT>
 int warpImage(DestImageT &destImage,
               SrcImageT const &srcImage,
-              geom::TransformPoint2ToPoint2 const & destToSrc,
```

Comment:
I'm not following why the directions are swapped for all these transforms.  Were they incorrect before?

Reply:
The version of warpImage that takes a Transform was modified to use the same direction as the one that takes an XYTransform because the Transform version was not yet in wide use and the XYTransform version was heavily used. This is mentioned in my community post about the changes https://community.lsst.org/t/replacing-wcs-and-tanwcs-with-skywcs/2531/2